src/base/Assert.hh

```diff
@@ -162,6 +167,15 @@
         }                                                \
     } while (0)
 
+#if !CELERITAS_USE_CUDA
```

Comment:
Why
```
#define X
#if !Y
#undef X
#define X
#endif
```
rather than
```
#if Y
#define X
#else
#define X
#endif
```

Reply:
Yeah, now that you mention it that would be more close to how the other macros are defined 😔 . I guess I did it to leave the macro closer to the documentation?

Reply:
> I guess I did it to leave the macro closer to the documentation?

By one line :~) :~)

Another "advantage" of the current version is that the effective version is intended one less time.